js/jquery.tablesorter.js

```diff
@@ -1328,6 +1328,11 @@
 				cache[ c.columns ].raw[ icell ] = tmp;
 				tmp = ts.getParsedText( c, cell, icell, tmp );
 				cache[ icell ] = tmp; // parsed
+				var tmpRow = cache[c.columns].$row;
+				if (tmpRow.length == 2) {
```

Comment:
Please change this to `if (tmpRow.length > 1) {` as it is possible to have more than one child row.